gems/smithy-client/lib/smithy-client.rb

```diff
@@ -26,6 +26,7 @@
 require_relative 'smithy-client/plugin'
 require_relative 'smithy-client/plugin_list'
 require_relative 'smithy-client/retry'
+require_relative 'smithy-client/service_error'
```

Comment:
Should `service_error` file be required prior to `dynamic_errors`? Since Dynamic error classes inherits from Service Error?

Reply:
It should not be necessary because a dynamic error is not created before loading files.